CondFormats/PPSObjects/xml/mapping_totem_nt2_2021.xml

```diff
@@ -5,101 +5,101 @@
 <top>
   <arm id="0">  <!-- sector 45, CMS negative z -->
     <nt2_plane id="0">
-      <nt2_tile id="0" hwId="0x206" FEDId="583" GOHId="3" IdxInFiber="6"/>
```

Comment:
Why there is a need to update 2021 XML file ?

Reply:
Yeah, you're right. I added a new 2023 XML with validity starting with the second T2 test file (first one was ~empty)